.travis.yml

```diff
@@ -6,10 +6,50 @@ php:
   - 5.5
   - 5.6
   - hhvm
+  - 7.0
 
 sudo: false
 
-install: composer install
+matrix:
+  fast_finish: true
+  include:
+    - php: '5.3'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '5.4'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '5.5'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '5.5'
+      env: SYMFONY_VERSION=2.8.*
+    - php: '5.5'
+      env: SYMFONY_VERSION=3.0.*
+    - php: '5.6'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '5.6'
+      env: SYMFONY_VERSION=2.8.*
+    - php: '5.6'
+      env: SYMFONY_VERSION=3.0.*
+    - php: 'hhvm'
+      env: SYMFONY_VERSION=2.3.*
+    - php: 'hhvm'
+      env: SYMFONY_VERSION=2.8.*
+    - php: 'hhvm'
+      env: SYMFONY_VERSION=3.0.*
+    - php: '7.0'
+      env: SYMFONY_VERSION=2.3.*
+    - php: '7.0'
+      env: SYMFONY_VERSION=2.8.*
+    - php: '7.0'
+      env: SYMFONY_VERSION=3.0.*
```

Comment:
Testing all combinations of Symfony and PHP is a bad idea. This makes the build matrix explode, and we are only allowed to have 5 concurrent jobs on Travis for the whole FOS organization. So creating 20 jobs for each build of FOSUserBundle is a no-go.